module/extract.go

```diff
@@ -65,14 +66,37 @@ func (v *tagExtractor) VisitField(f pgs.Field) (pgs.Visitor, error) {
 		v.tags[msgName] = map[string]*structtag.Tags{}
 	}
 
+
+	tags := structtag.Tags{}
+	tagLen := len(v.autoAddTags)
+	if tagLen > 1 || (tagLen == 1 && v.autoAddTags[0] != "") {
+		val := ToSnakeCase(v.Context.Name(f).String())
```

Comment:
Instead of defaulting to Snake case I would suggest taking the type of casing from the input as well. I would suggest snake_case, UpperCamelCase, lowerCamelCase to boot

Reply:
You mean when using the auto part ? It's a good idea, but should I support casing for each tag or a similar casing for all tags within default ?

Something like:
`auto="tag1+snake++tag2+upc++...++tagn"` with a default to snake if not provided ?

Reply:
Sorry for the extremely delayed response. I think it should be different for each tag. If you can give me access to your repo I'll make the necessary changes myself and complete the pr.

Reply:
I do not own the initial repository. I'll make the change. 
Ok to introduce a different case for each tag. (I also thinks its the cleanest).